zebedee-cms/src/main/java/com/github/onsdigital/zebedee/permissions/service/PermissionsServiceImpl.java

```diff
@@ -245,161 +154,34 @@ public void removeEditor(String email, Session session) throws IOException, Unau
-     * @param session         the session of the user that is attempting to set the viewer permissions.
-     * @throws IOException if reading or writing the access mapping fails.
-     * @throws UnauthorizedException if the users' session isn't authorised to edit collections.
-     */
-    @Override
-    public void setViewerTeams(Session session, String collectionID, Set<Integer> collectionTeams) throws IOException, UnauthorizedException {
-        if (session == null || !canEdit(session)) {
-            throw new UnauthorizedException(getUnauthorizedMessage(session));
-        }
-
-        if (collectionTeams == null) {
-            collectionTeams = new HashSet<>();
-        }
-
-        writeLock.lock();
-        try {
-            AccessMapping accessMapping = permissionsStore.getAccessMapping();
-            accessMapping.getCollections().put(collectionID, collectionTeams);
-            permissionsStore.saveAccessMapping(accessMapping);
-        } finally {
-            writeLock.unlock();
-        }
-    }
-
-    private boolean canEdit(String email, AccessMapping accessMapping) {
-        Set<String> digitalPublishingTeam = accessMapping.getDigitalPublishingTeam();
-        return (digitalPublishingTeam != null && digitalPublishingTeam.contains(standardise(email)));
-    }
-
-    private boolean canView(String email, String collectionId, AccessMapping accessMapping)
-            throws IOException {
-
-        // Check to see if the email is a member of a team associated with the given collection:
-        Set<Integer> teamIds;
-        readLock.lock();
-        try {
-            teamIds = accessMapping.getCollections().get(collectionId);
-        } finally {
-            readLock.unlock();
-        }
-        if (teamIds == null) {
-            return false;
-        }
-
-        return teamsServiceSupplier.getService().resolveTeams(teamIds).stream()
-                .anyMatch(team -> team.getMembers().contains(standardise(email)));
-    }
-
-    private String standardise(String email) {
-        return PathUtils.standardise(email);
-    }
-
     /**
      * User permission levels given an email
      *
      * @param email the user email
      * @return a {@link PermissionDefinition} object
-     * @throws IOException If a filesystem error occurs.
+     * @throws IOException           If a filesystem error occurs.
      * @throws UnauthorizedException If the request is not from an admin or publisher
      */
     @Override
     public PermissionDefinition userPermissions(String email, Session session) throws IOException,
             UnauthorizedException {
-        if ((session == null) || (!isAdministrator(session) && !isPublisher(session)
-                && !session.getEmail().equalsIgnoreCase(email))) {
+        if ((session == null) ||
+                !(isAdministrator(session) || isPublisher(session) || session.getEmail().equalsIgnoreCase(email))) {
```

Comment:
I think we do still because this is used on the user admin screen and the publishing officers can currently view the users and teams admin screens as read only.